components/MapView.js

```diff
@@ -77,6 +77,13 @@ var MapView = React.createClass({
     rotateEnabled: PropTypes.bool,
 
     /**
+     * If `true` the map will be cached to an Image for performance
+     * Default value is `false`.
+     *
+     */
+    cacheEnabled: PropTypes.bool,
```

Comment:
I see a potential solution for iOS as mentioned in http://stackoverflow.com/a/16186766. But currently I'm only using it in Android. Maybe sometime later when I have the time i'll work on this for iOS.